CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributor guidelines
+
+## Contributing
+
+In order to contribute to the project, the following steps should be followed:
+1. Fork this repository
+2. Create your feature branch
+3. Commit your local changes in the feature branch
+4. Push to your fork
+5. Create a pull Request
+
+Pull request are going to be reviewed by mainteners, some change can be requested before the Pull Request is accepted.
+
+## Development setup
+
+For building, we recommend to build an run inside a docker container. Refer to  [BUILDING.md](BUILDING.MD) for more information.
```

Comment:
For development purposes, we recommend building and running the project inside...